src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,68 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //ManufacturerService Test
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        System.out.println("Create manufacturers");
+        Manufacturer bmw = new Manufacturer("bmw", "Germany");
+        manufacturerService.create(bmw);
+
+        Manufacturer tesla = new Manufacturer("tesla", "USA");
+        manufacturerService.create(tesla);
+
+        Manufacturer volkswagen = new Manufacturer("volkswagen", "Germany");
+        manufacturerService.create(volkswagen);
+
+        System.out.println("Get all manufacturers");
+        manufacturerService.getAll().forEach(System.out::println);
+
+        System.out.println("Set country to manufacturer");
+        bmw.setCountry("Ukraine");
+        manufacturerService.update(bmw);
+
+        System.out.println(manufacturerService.get(bmw.getId()));
+
+        System.out.println("Delete manufacturer bmw");
+        manufacturerService.delete(bmw.getId());
+
+        System.out.println("Get all manufacturers after delete");
+        manufacturerService.getAll().forEach(System.out::println);
+
+        //DriverService Test
+
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+
+        System.out.println("Create drivers: ");
+        Driver misha = new Driver(null, "Mikhayl", "856512");
+        driverService.create(misha);
+
+        Driver andrew = new Driver(null, "Andrew", "85631321");
+        driverService.create(andrew);
```

Comment:
```suggestion
        Driver misha = new Driver("Mikhayl", "856512");
        driverService.create(misha);

        Driver andrew = new Driver("Andrew", "85631321");
        driverService.create(andrew);
```
the same here, create one more constructor without id